JSDemos/Demos/Charts/ServerSideDataProcessing/description.md

```diff
@@ -1 +1,9 @@
-In&nbsp;many cases, before being handed out to&nbsp;the client, data is&nbsp;processed on&nbsp;the server. The DevExtreme JavaScript UI components support this scenario. In&nbsp;this demo, the data source of&nbsp;the Chart component loads weather data for a&nbsp;selected month from an&nbsp;OData service. Each time the month is&nbsp;changed, the data source sends a&nbsp;new query to&nbsp;the service.
+In many cases, you need to process data on the server before the chart displays it. The Chart component supports this scenario. 
+
+In this demo, the data source of the Chart loads weather data for a selected month from an OData service. Each time you select a different month in the drop-down menu, the data source sends a new query to the service. To implement this functionality, assign a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object to the Chart [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
+
+In the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/), implement an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/). This demo shows how to [expand](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#expand) used values and how to [postProcess](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#postProcess) them.
+
+Set the [paginate](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#paginate) property to **false** to prevent data from partitioning. You can also apply [filter](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#filter) to the received values.
+
+Once you load the data, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type and its nested options: [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField), so the component can determine which object fields in the data source indicate Chart arguments and values.
```

Comment:
Please note that I already made this change in other PR at least once:

```suggestion
Once you load the data, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type and its nested options: [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField), so the component can determine the objects that indicate Chart arguments and values in the data source.
```

I'll just leave it here: https://tw.devexpress.devx/language-guidelines/misused-words/that-and-which/

Please be more attentive.